src/zenml/pipelines/pipeline_definition.py

```diff
@@ -474,7 +474,7 @@ def pipeline_(param_name: str):
     step_name()
 
 if __name__=="__main__":
-    pipeline_.with_options(config_file="config.yaml")(param_name="value2")
+    pipeline_.with_options(config_path="config.yaml")(param_name="value2")
```

Comment:
What's this ? Seems like a big change?

Reply:
It's fixing a typo in the docstring, i'll let you decide how big of a change that is :P

Reply:
haha oh i thought it was code!